### PR TITLE
Extend package_pam_pwquality_installed rule for RHEL

### PR DESCRIPTION
**Pull request by marcusburghardt**
Extend the `rule package_pam_pwquality_installed` to be also applicable for RHEL.

This rule checks if the package which brings the `pam_pwquality.so` module is installed.

### CODE REVIEW

File: shared/macros-bash.jinja [hunk 1]
```diff
@@ -638,7 +638,7 @@ zypper install -y "{{{ package }}}"
 {{%- if pkg_manager == "yum" or pkg_manager == "dnf" -%}}
 if rpm -q --quiet "{{{ package }}}" ; then
 {{% if SSG_TEST_SUITE_ENV %}}
-    {{{ pkg_manager }}} remove -y --setopt protected_packages= "{{{ package }}}"
+    rpm -e --nodeps "{{{ package }}}"
```
Comment (matejak) [inline shared/macros-bash.jinja:641]: Won't this break the test suite on non-rpm systems?

Comment (matusmarhefka) [reply shared/macros-bash.jinja:641]: Yes, here we can probably use similar approach as in https://github.com/ComplianceAsCode/content/blob/master/shared/macros-bash.jinja#L608

Comment (marcusburghardt) [reply shared/macros-bash.jinja:641]: No, it is included in the conditional for `yum` and `dnf` only. Distros with other package managers are not affected.

**Review by matusmarhefka — APPROVED** [review]
LGTM, thank you!

**Pull request merged**